### PR TITLE
Updated Todo MVC link

### PR DESCRIPTION
**Pull request by endelwar**

### CODE REVIEW

File: backbone.js/why-would-you-use-backbone/index.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ Backbone is an incredibly small library for the amount of functionality and stru
...
+* [Todo MVC - Todo list implemented in the many different types of frontend frameworks](http://todomvc.com/)
 
 
 ### Contributors
 
 * [FND](https://github.com/FND)
 
-__If you questions regarding why you should choose Backbone.js as your framework, please leave a comment below__
\ No newline at end of file
+__If you questions regarding why you should choose Backbone.js as your framework, please leave a comment below__
```
Comment (PeterDaveHello) [inline backbone.js/why-would-you-use-backbone/index.md]: Seems this diff has nothing to do with commit.

Comment (endelwar) [inline backbone.js/why-would-you-use-backbone/index.md]: yes, that's right, I'll remove it

**Comment by PeterDaveHello** [comment]
Same same, and would you please squash them into only one commit? Thanks.

**Comment by endelwar** [comment]
Removed new line at the end of file, it was added by github online editor

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**